DQMOffline/Trigger/plugins/TopMonitor.cc

```diff
@@ -0,0 +1,541 @@
+#include "DQMOffline/Trigger/plugins/TopMonitor.h"
```

Comment:
I think you should update the name of this class
unless you are going to add observables really TOP dependent